cloudify_agent/api/pm/base.py

```diff
@@ -274,6 +295,28 @@ def __init__(self, logger=None, **params):
                               backend=self.broker_url)
         self._celery.conf.update(
             CELERY_TASK_RESULT_EXPIRES=defaults.CELERY_TASK_RESULT_EXPIRES)
+        if self.broker_ssl_enabled:
+            self._celery.conf.BROKER_USE_SSL = {
+                'ca_certs': self._get_ssl_cert_path(),
+                'cert_reqs': ssl.CERT_REQUIRED,
+            }
+
+    def _get_celery_conf_path(self):
```

Comment:
as mentioned somewhere else, the location should probably be some place let intrusive than the site-packages dir that cloudify-agent lives in